### PR TITLE
Remove net452 from shims project

### PR DESCRIPTION
**Pull request by joaopgrassi**
Another project from #2138

* [x] `CHANGELOG.md` updated for non-trivial changes
* [x] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Shims.OpenTracing/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Removes .NET Framework 4.5.2 support. The minimum .NET Framework version
+  supported is .NET 4.6.1. ([2138](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138))
```
Comment (reyang) [inline src/OpenTelemetry.Shims.OpenTracing/CHANGELOG.md:6]:
```suggestion
  supported is .NET 4.6.1. ([#2138](https://github.com/open-telemetry/opentelemetry-dotnet/issues/2138))
```

**Review by reyang — APPROVED** [review]
LGTM.

File: src/OpenTelemetry.Shims.OpenTracing/OpenTelemetry.Shims.OpenTracing.csproj [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (cijothomas) [inline src/OpenTelemetry.Shims.OpenTracing/OpenTelemetry.Shims.OpenTracing.csproj:3]: Please add net461 here.

**Review by cijothomas — CHANGES_REQUESTED** [review]
Please add net461 as there is no other netframework target here.

File: src/OpenTelemetry.Shims.OpenTracing/OpenTelemetry.Shims.OpenTracing.csproj (hunk 2 above)
Comment (joaopgrassi) [reply src/OpenTelemetry.Shims.OpenTracing/OpenTelemetry.Shims.OpenTracing.csproj]: Done.

**Comment by joaopgrassi** [comment]
I think I fixed the remaining the issues @cijothomas.

Just something I noticed: Is there a reason why the package `Microsoft.CodeAnalysis.PublicApiAnalyzers` is only added on Windows? I ran a build/test locally and didn't get any errors because I use Linux, but then I saw the CI failing..

In order to generate the public api files, I had to manually add the package and run `dotnet format -a warn`. Since all worked fine, I wonder if we can't just always add the package? Then people would not forgot to add like I did. It's the first time I use this, so just asking to understand it.

**Review by cijothomas — APPROVED** [review]

**Comment by cijothomas** [comment]
> I think I fixed the remaining issues @cijothomas.
> 
> Just something I noticed: Is there a reason why the package `Microsoft.CodeAnalysis.PublicApiAnalyzers` is only added on Windows? I ran a build/test locally and didn't get any errors because I use Linux, but then I saw the CI failing..
> 
> In order to generate the public api files, I had to manually add the package and run `dotnet format -a warn`. Since all worked fine, I wonder if we can't just always add the package? Then people would not forget to add the api files like I did. It's the first time I use this, so just asking to understand it.

I don't recollect why... If you can make PR to do it in both win/linux, and if that makes its easy for linux based devs to catch issues easily, we can take it. Thanks!

**Pull request merged**